src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationFactoryClientOptions.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET  Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using Microsoft.AspNetCore.Mvc.Testing.Handlers;
+
+namespace Microsoft.AspNetCore.Mvc.Testing
+{
+    /// <summary>
+    /// The default options to use to when creating
+    /// <see cref="HttpClient"/> instances by calling
+    /// <see cref="WebApplicationFactory{TEntryPoint}.CreateClient(WebApplicationFactoryClientOptions)"/>.
+    /// </summary>
+    public class WebApplicationFactoryClientOptions
```

Comment:
@Eilon This class (along with a property on WebApplicationFactory) controls the behavior of CreateClient()